src/crypto/tls/cfkem.go

```diff
@@ -31,17 +31,17 @@ import (
 type clientKeySharePrivate interface{}
 
 var (
-	Kyber512X25519 = CurveID(0xfe30)
-	Kyber768X25519 = CurveID(0xfe31)
+	Kyber512X25519Draft1 = CurveID(0xfe30)
+	Kyber768X25519Draft1 = CurveID(0xfe31)
```

Comment:
* Does `Draft1` refer to an I-D that defines these code points? If so we should add a link to the draft here.
* Typically we would write this `Draf01`.

Reply:
+1 -- it would be great if these were actually the codepoint values advertised in [the hybrid KEX spec](https://datatracker.ietf.org/doc/draft-ietf-tls-hybrid-design/). @bwesterb, could you please reach out to Douglas et al. to see if they're willing to add these?

Reply:
Agreed with Douglas on the name.